src/components/CodeView/index.spec.tsx

```diff
@@ -250,34 +261,47 @@ describe(__filename, () => {
 
   it('renders a link for each line number', () => {
     const location = createFakeLocation();
-    const root = renderWithLinterProvider({ content: 'single line', location });
+    const { renderContent, shell } = simulateCommentableLine({
```

Comment:
I'm finding the name `shell` to be a bit confusing. I've gotten used to it seeing it throughout the file, but I wonder if we could come up with a better naming. What if instead of `shell` you used `root`, because `root` is really what we used to call this (and call a similar element in other tests), and then, instead of using `root` to represent a rendered line of code, call that `line` instead. I see you did that at line 224 of this file. I think those names would be easier to understand.

I realize that changing that wholesale means having to replace some instances of `root` with `line` in some of the earlier tests, yielding a larger diff, but I think that would be worth it.

Reply:
Sure, I can do that